x-pack/plugins/security_solution/server/endpoint/routes/trusted_apps/service.ts

```diff
@@ -120,8 +167,18 @@ export const updateTrustedApp = async (
     throw new TrustedAppNotFoundError(id);
   }
 
-  // Validate update TA entry - error if not valid
-  // TODO: implement validations
+  const unexistingPolicies = await getUnexistingPoliciesFromTrustedApp(
+    savedObjectClient,
+    packagePolicyClient,
+    updatedTrustedApp
+  );
+
+  if (!isEmpty(unexistingPolicies)) {
```

Comment:
some of these lodash methods crack me up. you could just `!unexistingPolicies.length`
(don't get me wrong - that method is helpful with objects, but array's are pretty straight forward 😃 )

Reply:
Yeah, I like to use lodash in this cases just because it gets more readable (at least for me). Do you think it has some performance involvement?